save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityProjectWindow.kt

```diff
@@ -21,100 +22,203 @@ import react.useState
 import web.cssom.ClassName
 import web.html.ButtonType
 
+import kotlinx.serialization.json.JsonObject
+
 /**
  * Component that allows to add new project in vulnerability
  */
 val vulnerabilityProjectWindow: FC<VulnerabilityProjectWindowProps> = FC { props ->
+
+    @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
```

Comment:
please do not suppress, fix it
val emptyRange: Range<JsonObject> = Range(

Reply:
Fixed